proto/vtctldata.proto

```diff
@@ -1303,6 +1359,68 @@ message ValidateVSchemaResponse {
   map<string, ValidateShardResponse> results_by_shard = 2;
 }
 
+message WorkflowDeleteRequest {
+  string keyspace = 1;
+  string workflow = 2;
+  bool keep_data = 3;
+  bool keep_routing_rules = 4;
+}
+
+message WorkflowDeleteResponse {
+  message TabletInfo {
+    string tablet = 1;
+    // The value is based on the query result's RowsAffected
+    // being 0 or not.
+    bool deleted = 2;
+  }
+  string summary = 1;
+  repeated TabletInfo details = 2;
+}
+
+message WorkflowStatusRequest {
+  string keyspace = 1;
+  string workflow = 2;
+}
+
+message WorkflowStatusResponse {
+  message TableCopyState {
+    int64 rows_copied = 1;
+    int64 rows_total = 2;
+    int32 rows_percentage = 3;
```

Comment:
IIRC I was basing this on the types already being used in the wrangler. Makes sense to try to switch to floats. But I also can't refactor everything in this one PR or it will be many months before it's done and we then may also need to make the vtctldclient implementation experimental. These are absolution critical and fragile / susceptible to subtle bugs. And when vreplication breaks people have outages, lose data, etc. So please keep this in mind when you suggest various refactorings. :-) Not that this particular one is huge, but in total these are adding up to a lot. Not only that, but for N years we will then have two parallel implemenations of things that we have to find and fix bugs in. This is not a small thing.